meta-sokol-flex-staging/recipes-core/glibc/ldconfig-multilib.bb

```diff
@@ -0,0 +1,34 @@
+SUMMARY = "Extra ld.so configuration files for multilib library paths"
+DESCRIPTION = "${SUMMARY}"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COREBASE}/meta/COPYING.MIT;md5=3da9cfbcb788c80a0384361b4de20420"
+INHIBIT_DEFAULT_DEPS = "1"
+
+deltask do_fetch
+deltask do_unpack
+deltask do_patch
+deltask do_configure
+deltask do_compile
+deltask do_populate_sysroot
+
+python do_install () {
+    """Install /etc/ld.so.conf.d/<defaulttune>.conf for all multilibs."""
+    from pathlib import Path
+
+    ldsodir = Path(d.getVar('D')) / d.getVar('sysconfdir')[1:] / 'ld.so.conf.d'
+    ldsodir.mkdir(parents=True, exist_ok=True)
+
+    libdir_vars = ['base_libdir', 'libdir']
+    for multilib, mldata in per_multilib(d):
+        libdirs = sorted(set(mldata.getVar(var) for var in libdir_vars))
+        defaulttune = mldata.getVar('DEFAULTTUNE')
+        fname = '{1}.conf'.format(defaulttune)
```

Comment:
@kergoth  getting this error

ERROR: ldconfig-multilib-1.0-r0 do_install: Error executing a python function in exec_func_python() autogenerated:

The stack trace of python calls that resulted in this exception/failure was:
File: 'exec_func_python() autogenerated', lineno: 2, function: <module>
     0001:
 *** 0002:do_install(d)
     0003:
File: '/var/jenkins/workspace/mel_ginkgo_async1_main_incremental/buildtype/mel/label/flex-u22/machine/imx8mpevk-flex/repotop/meta-sokol-flex/meta-sokol-flex-staging/recipes-core/glibc/ldconfig-multilib.bb', lineno: 25, function: do_install
     0021:    libdir_vars = ['base_libdir', 'libdir']
     0022:    for multilib, mldata in per_multilib(d):
     0023:        libdirs = sorted(set(mldata.getVar(var) for var in libdir_vars))
     0024:        defaulttune = mldata.getVar('DEFAULTTUNE')
 *** 0025:        fname = '{1}.conf'.format(defaulttune)
     0026:        with open(ldsodir / fname, 'w') as f:
     0027:            f.writelines(l + '\n' for l in libdirs)
     0028:}
     0029:
Exception: IndexError: Replacement index 1 out of range for positional args tuple

Reply:
Fixed, thanks! No idea how I let that slip in.